source/administration/changelog.md

```diff
@@ -4,6 +4,127 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
 
 Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.33 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2021-03-16**
+
+### Breaking Changes
+ - Deleting a reaction is now a soft delete in the ``Reactions`` table. A schema update is required and may take up to 15 seconds on first run with large data sets.
+ - Improved the websocket implementation by using epoll to manually read from a websocket connection. As a result, the number of goroutines is expected to go down by half. This implementation is only available on Linux and FreeBSD-based distributions. If you are using NGINX as a proxy to Mattermost, please ensure to have ``proxy_http_version 1.1;`` in the block that handles the websocket path.
```

Comment:
Not sure I understand what is the breaking change here?

"Improved the websocket implementation by using epoll to manually read from a websocket connection. As a result, the number of goroutines is expected to go down by half. This implementation is only available on Linux and FreeBSD-based distributions."

Perhaps we need to rephrase this bullet to identify the breaking change and why it exists

Reply:
@streamer45 Please help with details here.

Reply:
Maybe that goes a bit into too much detail for a release note. The gist is that we improved the performance of handling websocket connections on servers running on Linux/Unix platforms (more or less anything except Windows). 

As part of this we discovered that we were suggesting incorrect NGINX configurations that will break websocket connectivity if used with the new release so the only action required is for admins deploying Mattermost behind NGINX. If that's the case they should review and possibly update their NGINX configuration by adding the suggested line to the websocket related block. 

cc @agnivade

Reply:
The breaking change is only from an nginx perspective. The new code changes require nginx to send the http version header, whereas previous versions did not. The first two sentences don't really explain the need for the breaking change. It's better for a release note. I would rephrase to something like:

```
A new websocket implementation requires the HTTP version to be 1.1. If you are using NGINX as a proxy to Mattermost, please ensure to have ``proxy_http_version 1.1;`` in the block that handles the websocket path.
```

Reply:
I've added the suggested note in the breaking changes section, and moved the first two sentences to the Websocket section below.